backend/molgenis-emx2-rdf/src/main/java/org/molgenis/emx2/rdf/RDFService.java

```diff
@@ -148,24 +153,29 @@ public void describeAsRDF(
       final Schema... schemas) {
     try {
       final ModelBuilder builder = new ModelBuilder();
-      builder.setNamespace("rdf", NAMESPACE_RDF);
-      builder.setNamespace("rdfs", NAMESPACE_RDFS);
-      builder.setNamespace("xsd", NAMESPACE_XSD);
-      builder.setNamespace("owl", NAMESPACE_OWL);
-      builder.setNamespace("sio", NAMESPACE_SIO);
-      builder.setNamespace("qb", NAMESPACE_QB);
-      builder.setNamespace("skos", NAMESPACE_SKOS);
-      builder.setNamespace("dcterms", NAMESPACE_DCTERMS);
-      builder.setNamespace("dcat", NAMESPACE_DCAT);
-      builder.setNamespace("foaf", NAMESPACE_FOAF);
-      builder.setNamespace("vcard", NAMESPACE_VCARD);
-      builder.setNamespace("org", NAMESPACE_ORG);
-      builder.setNamespace("fdp-o", NAMESPACE_ORG);
 
+      // Defines if all used schemas have a custom_rdf setting.
+      boolean allIncludeCustomRdf = true;
       // Define the schemas at the start of the document.
       for (final Schema schema : schemas) {
         final Namespace ns = getSchemaNamespace(schema);
         builder.setNamespace(ns);
+        // Adds custom RDF to model.
+        if (schema.hasSetting(SETTING_CUSTOM_RDF)) {
```

Comment:
hmm, per schema means they all add up when multiple schemas? that is then also no problem

Reply:
Exactly. Current behavior simply combines all `custom_rdf` data if using multiple schemes. Special cases:
* If one or more has no `custom_rdf`, default list is added (as a scheme assumes these prefixes are present)
* If schemes have an empty `custom_rdf` instead of non-existing, this should cause the default addition to be skipped.